README.md

```diff
@@ -48,20 +49,42 @@ create_table :emails do |t|
  end
 ```
 
+### Configure custom association names
+
+You may not want to use the association `user.emails` or `email.users`. You can customize the name of the associations used.
+
+```ruby
+# Default is :user for Email model
+Devise::MultiEmail.parent_association_name = :team
```

Comment:
Possible to make this a standard config form ?  Would be great if it's something like:

```
Devise::MultiEmail.configure do |config|
  config.parent_association_name = :team
end
```

It's more conventional as well.